### ISSUE TITLE
Consider releasing alpha releases to conda-forge dev label?

### ISSUE THREAD

**Issue by tlambert03** [conda-forge/cython-feedstock#84]
Hello!  Just wondering whether it would be possible to release one of the alpha versions on conda-forge.  I have a [package](https://github.com/conda-forge/psygnal-feedstock/pull/7) that I'd like to build for conda-forge, but it uses a fair amount of generics and it seems I need 3.0a7 or later.

I asked some conda forge folks, they pointed to https://github.com/conda-forge/lief-feedstock/pull/23 as an example of how this was done for `lief`.

Is this something you'd consider for cython?

**Comment by ocefpaf** [comment]
We can do that following https://github.com/conda-forge/cfep/blob/main/cfep-05.md
Do you want to send a PR for that?

**Comment by tlambert03** [comment]
will give it a try!  thanks

**Comment by jaimergp** [comment]
Try now @tlambert03 !

**Comment by jakirkham** [comment]
AIUI the work in that PR has been primarily focused on making sure all the tests are passing. It may be possible (and in fact reasonable) to build the dev package anyways even with failing tests to enable more people to try.

**Comment by h-vetinari** [comment]
Sure, that's easily possible. I only added the tests to #75 as a feedback mechanism to the cython devs; it's not necessary.

Now that there's a dev branch, I can rebase the PR against that.

**Comment by jakirkham** [comment]
Thanks Axel! Also thanks for all of your hard work in that PR 😄

**Comment by h-vetinari** [comment]
It seems @tlambert03 pushed to `dev` already - was that intentional (as opposed to raising a PR)?

**Comment by h-vetinari** [comment]
Ah nevermind, just saw #86. Guess this issue can be closed then?

For the record - without the test suite updates - the only pertinent changes from #75 would have been:
```
diff --git a/recipe/meta.yaml b/recipe/meta.yaml
index 0a5175e..255bcc1 100644
--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -5,8 +5,8 @@ package:
   version: {{ version }}

 source:
-  url: https://pypi.io/packages/source/C/Cython/Cython-{{ version }}.tar.gz
-  sha256: 342e95121a3d1a67cbcf7b340391eb40cc5ce3d2a79d7873e005e8783353d89d
+  url: https://github.com/cython/cython/archive/refs/tags/{{ version }}.tar.gz
+  sha256: d530216e015fd365bf3327a176e0073d0e5b8d48781f387336459f10032d776f

 build:
   number: 0
```

**Comment by tlambert03** [comment]
yep, this can be closed now, and #75 can keep working on tests?  Thanks all!

**Comment by jakirkham** [comment]
Yeah sorry missed that as well. Was still going through my notifications. Saw PR ( https://github.com/conda-forge/cython-feedstock/pull/85 ), which was closed, first so thought maybe the effort was abandoned. Glad to see that is not the case 😌
